wagtail_footnotes/templatetags/wagtailfootnotes_tags.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def get_reference_ids(value, footnote_uuid):
+    """This takes the current `footnote_uuid` and returns the list of references in the page content to that footnote.
+    This template tag is only necessary because it is not possible to do dictionary lookups using variables as keys in
+    Django templates.
+    """
+    if hasattr(value, "footnotes_references"):
+        return value.footnotes_references.get(footnote_uuid, [])
+    return []
```

Comment:
I am no sure we need this. if we drop `footnotes_list`, we can update the template to check for `page.footnotes_references` and tweak the loops accordingly